includes/wf_crm_webform_base.inc

```diff
@@ -44,6 +44,20 @@ abstract class wf_crm_webform_base {
           $this->_contribution_page = wf_civicrm_api('contribution_page', 'getsingle', array('id' => $contribution_page_id));
         }
         return $this->_contribution_page;
+        
+      case 'tax_rate':
+        $taxSettings = CRM_Core_BAO_Setting::getItem(CRM_Core_BAO_Setting::CONTRIBUTE_PREFERENCES_NAME, 'contribution_invoice_settings');
+        if (isset($taxSettings['invoicing'])) {
+          if ($taxSettings['invoicing']) {
```

Comment:
Since webform_civicrm v4 supports CiviCRM versions 4.4 and up, this setting may not exist, therefore the variable `$taxSettings` may be NULL or an empty string, so it might not be safe to treat it as an array (php 5.3 is especially quirky about treating non-arrays as arrays).
I would suggest the above two if statements be combined into a single if statement as follows:

    if (is_array($taxSettings) && !empty($taxSettings['invoicing'])) {